applications/asset_tracker_v2/src/modules/Kconfig.modem_module

```diff
@@ -9,6 +9,7 @@ menuconfig MODEM_MODULE
 	select LTE_LINK_CONTROL
 	select PDN
 	select PDN_ESM_STRERROR
+	select PDN_EMM_STRERROR
```

Comment:
These are 2KB of flash each. We are running out of flash with some ATv2 configurations. Should we enable these in debug overlay rather than always? I don't have an exact answer but just a thought as there are memory limitations.

@simensrostad

Reply:
It's possible to save a bit more than 100 bytes of flash from each table by changing the `reason` field to an `uint8_t` and making the struct packed. But it's only a small improvement.

Reply:
@trantanen Its probably best to guard these tables come to think about it. Having them in enabled by default in the debug overlay sounds like a good idea.